src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -75,7 +80,7 @@ namespace winrt::TerminalApp::implementation
         GETSET_PROPERTY(bool, ForceFullRepaintRendering, false);
         GETSET_PROPERTY(bool, SoftwareRendering, false);
         GETSET_PROPERTY(bool, ForceVTInput, false);
-        GETSET_PROPERTY(bool, DebugFeaturesEnabled); // default value set in constructor
+        GETSET_PROPERTY(bool, DebugFeaturesEnabled, debugFeaturesDefault);

```

Comment:
please keep this in the constructor. it avoids leaking a global named `debugFeaturesDefault` into any file that includes GAS.h